main.go

```diff
@@ -1028,6 +1034,20 @@ func parseFlags() (config, error) {
 		cfg.traces.writeEndpoint = rawTracesWriteEndpoint
 	}
 
+	if cfg.traces.readTemplateEndpoint != "" {
+		if cfg.traces.readEndpoint != nil {
```

Comment:
do we also need to check for `readTemplateEndpoint` [here](https://github.com/observatorium/api/blob/main/main.go#L1009-L1018) ? (to avoid having 2 flags defined as it is done here)

Reply:
The initial version of the PR had only one flag.  The PR was simpler.  I broke the behavior into flags because of the suggestion of @squat https://github.com/observatorium/api/pull/271#discussion_r840353683 who wanted it to be easy to remove this behavior in the future.

Reply:
ah, I mean if we also need to check for `readTemplateEndpoint` at https://github.com/observatorium/api/blob/main/main.go#L1009-L1018 and throw `only one of --traces.read.endpoint and --experimental.traces.read.endpoint-template allowed` in case `readTemplateEndpoint` it is defined too

Reply:
ping @esnible